src/functions/simple.cc

```diff
@@ -102,6 +102,32 @@ napi_value bind_OCGetServerInstanceIDString(napi_env env,
   C2J_SET_RETURN_VALUE(env, info, string_utf8, uuid, strlen(uuid));
 }
 
+napi_value bind_OCSetDeviceId(napi_env env, napi_callback_info info)
+{
+    OCUUIdentity    *ocid = NULL;
+    OCStackResult   res = OC_STACK_INVALID_PARAM;
+    char            byte[2] = {0};
+
+    J2C_DECLARE_ARGUMENTS(env, info, 1);
+    J2C_GET_STRING_TRACKED_JS_THROW(uuid, env, arguments[0], true, "uuid");
```

Comment:
Why not simply use `false` and then the function will return if `uuid` is `NULL`, and the if-statement below is unnecessary?